.changeset/short-plants-own.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": minor
+---
+
+Added `error` and `warning` status options to TrackerStep's `state` prop.
```

Comment:
should you also mention the new `icon` prop here?

Reply:
Ah, good spot! I'd missed that. Fixed now